doc/content/modules/user-manual/pages/release-notes/2025.2.0.adoc

```diff
@@ -126,6 +126,7 @@ image::release-notes-diagrams-export-as-png.png[Export diagram as PNG, width=50%
 * `Concern Usage`
 * `Stakeholder Membership`
 
+-  Display in the _Details_ view the features _isDefault_ and _isInitial_ in Core tab of `FeatureValue` concept.
```

Comment:
"feature" is more a concept known by EMF users. I would prefer the term "properties" please.
It also allows to not confuse with the SysMLv2 concept of `Feature`.